core/src/main/java/org/infinispan/marshall/core/GlobalMarshaller.java

```diff
@@ -31,124 +35,118 @@
  * @since 5.0
  */
 @Scope(Scopes.GLOBAL)
-public class GlobalMarshaller extends AbstractMarshaller implements StreamingMarshaller {
+public class GlobalMarshaller implements StreamingMarshaller {
 
    private static final Log log = LogFactory.getLog(GlobalMarshaller.class);
    private final boolean trace = log.isTraceEnabled();
 
-   private JBossMarshaller defaultMarshaller;
+   GlobalComponentRegistry gcr;
+   RemoteCommandsFactory cmdFactory;
 
-   private ExternalizerTable extTable;
-   private GlobalConfiguration globalCfg;
+   InternalMarshaller internal;
```

Comment:
The `GlobalMarshaller` seems to just delegate to the `InternalMarshaller`, couldn't we implement the methods directly in `GlobalMarshaller`?